src/quantum/HISTORY.rst

```diff
@@ -3,6 +3,15 @@
 Release History
 ===============
 
+0.15.0
+++++++
+* [2022-04-22] Version intended to work with QDK version v0.25.n (Release was done prior to the full QDK release.)
```

Comment:
Let's change the version to 'v0.24.206417' which is the RC build and remove the clarification in parenthesis. If there is a newer version, we can always update the history offline. Also, for date, I recommend 2022-04-25, since the earliest this will go out is Monday morning Shanghai time.

Reply:
Made those changes.  Thanks, Ricardo!